libraries/AP_Scripting/examples/ahrs-set-origin-to-home-location.lua

```diff
@@ -0,0 +1,14 @@
+-- example script for using "set_origin()"
+-- sets the ekf origin to the home location every 5 seconds
+
+function update ()
+    
+    home = ahrs:get_home()
+    origin = ahrs:get_origin()
+
+    ahrs:set_origin(home)
```

Comment:
This script won't actually work though I think because the origin can only be set once and home can only be set once the origin has been set.  It would probably be better to make the script set the origin to some specific lat, lon, alt.. perhaps somewhere near the SITL testing location in Canberra.

Reply:
I did test this in SITL and it seems to work. I'm able to set home repeatedly and the origin follows. I was able to see it quite nicely by running the "print-home-and-origin" script at the same time.

Reply:
@CallanDaniel, ok, that doesn't sound good though actually.  It should not be possible to update the origin after it has been set.

Reply:
Understood. I just wrote the scripts to see if the bindings were working, but didn't really think about the context of changing the origin in flight.